src/test/ui/privacy/public-instead-of-pub.rs

```diff
@@ -0,0 +1,4 @@
+// Checks what happens when `public` is used instead of the correct, `pub`
+// edition:2018
+public struct MyStruct;
+//~^ ERROR 3:8: 3:14: expected one of `!` or `::`, found keyword `struct`
```

Comment:
```suggestion
//~^ ERROR 3:8: 3:14: expected one of `!` or `::`, found keyword `struct`
//~^^ HELP write `pub` instead of `public` to make the item public
```
nit: we could check for the suggestion that we're interested in here too